server/game/cards/18-FH/LeftHandLucasCodd.js

```diff
@@ -5,7 +5,7 @@ class LeftHandLucasCodd extends DrawCard {
     setupCardAbilities() {
         this.reaction({
             when: {
-                onDefendersDeclared: event => event.player !== this.controller && event.numOfDefendingCharacters === 0 && this.hasAttackingRaider()
+                onDefendersDeclared: event => event.player !== this.controller && event.numOfDefendingCharacters === 0 && this.hasAttackingRaider(this.controller)
```

Comment:
This looks equivalent to what was there before. `this.controller` is being passed in instead of hardcoded in the method. What am I missing?

Reply:
You are missing nothing, i had a look at the commit history of LeftHandLucasCodd in the playtesting repo to figure out why this diff exists:
there is another card with the same effect coming up in the next pack and it introduced a bug to LeftHandLucasCodd, so this is what the correction looks like.
I will change it back and also update the new card which should work with the same solution.